core/src/main/java/org/elasticsearch/index/query/FieldMaskingSpanQueryBuilder.java

```diff
@@ -30,17 +35,37 @@
 
     private final SpanQueryBuilder queryBuilder;
 
-    private final String field;
+    private final String fieldName;
 
-    private float boost = -1;
+    private float boost = 1.0f;
 
     private String queryName;
 
     static final FieldMaskingSpanQueryBuilder PROTOTYPE = new FieldMaskingSpanQueryBuilder(null, null);
 
-    public FieldMaskingSpanQueryBuilder(SpanQueryBuilder queryBuilder, String field) {
+    /**
+     * Constructs a new {@link FieldMaskingSpanQueryBuilder} given an inner {@link SpanQueryBuilder} for
+     * a given field
+     * @param queryBuilder inner {@link SpanQueryBuilder}
+     * @param fieldName the field name
+     */
+    public FieldMaskingSpanQueryBuilder(SpanQueryBuilder queryBuilder, String fieldName) {
         this.queryBuilder = queryBuilder;
-        this.field = field;
+        this.fieldName = fieldName;
+    }
+
+    /**
+     * @return the field name for this query
+     */
+    public String fieldName() {
+        return this.fieldName;
+    }
+
+    /**
+     * @return the inner {@link QueryBuilder}
+     */
+    public SpanQueryBuilder innerQuery() {
```

Comment:
Good question. I just checked, in BoostingQB we use `QueryBuilder positive()` and `QueryBuilder negative()`, but that was mainly because there were setters called `positiv(QB)` and `negative(QB)`. In ConstantScoreQB the getter is simply `query()`. In the QueryFilterBuilder PR I use `innerQueryBuilder()` but since at some point we might move away from the builders for the intermediate query representation, I'd rather change that there as well.

So I'd vote for `innerQuery()` or simply `query()` if there is only one query that is beeing wrapped. No strong preferences there, maybe leaning to `innerQuery()` because when using it like `newFilterQuery.innerQuery()` it's clearer to me that the output doesn't refer to the parent query itself.